addons/mail/static/src/models/channel_member.js

```diff
@@ -57,6 +57,10 @@ registerModel({
             readonly: true,
             required: true,
         }),
+        invitingThread: one('Thread', {
+            inverse: 'invitedMembers',
+            isCausal: true,
+        }),
```

Comment:
field not used an isCausal is wrong here (should not delete thread on deleting member)